saleor/graphql/order/tests/benchmark/fixtures.py

```diff
@@ -1,12 +1,28 @@
+import random
 import uuid
 
 import pytest
 from prices import Money, TaxedMoney
 
 from .....account.models import User
 from .....order.models import Order
+from .....payment import ChargeStatus
+from .....payment.models import Payment
 
 ORDER_COUNT_IN_BENCHMARKS = 10
+PAYMENTS_PER_ORDER = 3
+
+
+def _create_payments_for_order(order):
```

Comment:
The name of the function is a little misleading as this function does not create payments in the database. Maybe something like `_prepare_payments_for_order`?